src/generate.js

```diff
@@ -16,12 +16,14 @@ self.generate = function(options) {
 	if (!options.hasOwnProperty('symbols')) options.symbols = false;
 	if (!options.hasOwnProperty('uppercase')) options.uppercase = true;
 	if (!options.hasOwnProperty('excludeSimilarCharacters')) options.excludeSimilarCharacters = false;
+	if (!options.hasOwnProperty('repeatCharactersLimit')) options.repeatCharactersLimit = 0;
 
 	var lowercase = 'abcdefghijklmnopqrstuvwxyz',
 		uppercase = 'ABCDEFGHIJKLMNOPQRSTUVWXYZ',
 		numbers = '0123456789',
 		symbols = '!@#$%^&*()+_-=}{[]|:;"/?.><,`~',
-		similarCharacters = /[ilLI|`oO0]/g;
+		similarCharacters = /[ilLI|`oO0]/g,
+		repeatingCharacters = options.repeatCharactersLimit ? new RegExp('(.)\\1{' + (options.repeatCharactersLimit) + ',}', 'g') : undefined;
```

Comment:
Do you mean, if a user passes in `1` as the repeatCharactersLimit option, then throw an error?

It's been a while since I worked on this. I'll re-familiarize myself with it again, to have a better understanding of passing in `0` & `1` with this option. I forgot the conclusions I came up with on those use cases.